src/controllers/data-removal.js

```diff
@@ -1,9 +1,13 @@
 import { mainLayout } from '../views/main.js'
 import { dataRemoval } from '../views/partials/data-removal.js'
+import { userMenu } from '../views/partials/user-menu.js'
 
 function dataRemovalPage (req, res) {
+  const fxaProfileData = req.user.fxa_profile_json
+
   const data = {
-    partial: dataRemoval
+    partial: dataRemoval,
+    userMenu: userMenu(fxaProfileData)
```

Comment:
We should come up with a better way to pass this data.  I don't have the answer yet (it's getting late!) but let's have a think about this to avoid needing to pass this for every partial.

Reply:
I agree. I didn’t want to blow up the scope of this PR, but thought about how to handle passing multiple partials could be done best. I created PR with a suggestion and for discussing possible approaches:
- #2761